generator/generator-opc-generator/__tests__/app.js

```diff
@@ -0,0 +1,16 @@
+'use strict';
+const path = require('path');
+const assert = require('yeoman-assert');
+const helpers = require('yeoman-test');
+
+describe('generator-opc-generator:app', () => {
+  beforeAll(() => {
+    return helpers
+      .run(path.join(__dirname, '../generators/app'))
+      .withPrompts({ someAnswer: true });
```

Comment:
update the tests for the generator. These are not relevant to the generator we are working on

Reply:
Added the test cases to check if the files exists which would be used in the generator

Reply:
@hybridx hey i still see test  `.withPrompts({ someAnswer: true });` you need to replace it with the values that we need in the prompt